lib/puppet/provider/package/pip.rb

```diff
@@ -31,7 +31,11 @@ def self.instances
     packages = []
     pip_cmd = self.pip_cmd
     return [] unless pip_cmd
-    execpipe "#{pip_cmd} freeze" do |process|
+    command = [pip_cmd, 'freeze']
+    if Puppet::Util::Package.versioncmp(self.pip_version, '8.1.0') >= 0 # a >= b
+        command << '--all'
```

Comment:
It will be against the base branch, in this case master. Looks like the pip spec has changed, probably due to us merging https://github.com/puppetlabs/puppet/pull/5903 this morning.